src/cgame/cg_ui.h

```diff
@@ -40,6 +40,9 @@ struct rectDef_t
 const char* Trans_Gettext( const char *msgid ) PRINTF_TRANSLATE_ARG(1);
 const char* Trans_Pgettext( const char *ctxt, const char *msgid ) PRINTF_TRANSLATE_ARG(2);
 const char* Trans_GettextPlural( const char *msgid, const char *msgid_plural, int num ) PRINTF_TRANSLATE_ARG(1);
+void Trans_SetLanguage( const char* lang );
+void Trans_UpdateLanguage_f();
```

Comment:
I'd say it probably should be refactored anyway in the end. I think it's good like this, as I've made `Trans_SetLanguage( const char* lang )` private to avoid wrong uses.